src/pages/passengerType.tsx

```diff
@@ -72,16 +102,23 @@ const PassengerType = ({ errors = [], csrfToken }: PassengerTypeProps): ReactEle
     </TwoThirdsLayout>
 );
 
-export const getServerSideProps = (ctx: NextPageContextWithSession): { props: PassengerTypeProps } => {
+export const getServerSideProps = async (ctx: NextPageContextWithSession): Promise<{ props: PassengerTypeProps }> => {
     const csrfToken = getCsrfToken(ctx);
     const passengerTypeAttribute = getSessionAttribute(ctx.req, PASSENGER_TYPE_ATTRIBUTE);
 
+    let savedGroups = getSessionAttribute(ctx.req, SAVED_PASSENGER_GROUPS);
+    if (savedGroups === undefined) {
```

Comment:
does !savedGroups equate to the same thing?

Reply:
!savedGroups might match things that are "falsy" but actually set. The undefined ensures that it is genuinely not set at this point and not just falsy